apps/docs/content/components/Pillar/index.mdx

```diff
@@ -71,18 +89,18 @@ Cards should be used for noteworthy information to immediately draw user attenti
 
 #### Group size
 
-Cards should be used in small groups (3, 4 or 5 items maximum), whereas Pillars can be used in larger (more than 5 items) or smaller groups. As users encounter an increasing number of items within a group, the distinctiveness of each individual element diminishes.
+Cards should be used in small groups of three to five items, while Pillars can accommodate larger groups of up to six items. The more items in a group, the less distinctive each individual element becomes.
```

Comment:
I thought the six Pillar limit was only for Pillars with images? AFAIK we should still be able to do up to 9 without  (3x3 grid on desktop).

Reply:
You're right, good catch!